Gemfile.lock

```diff
@@ -4,65 +4,69 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.6)
```

Comment:
avoid making changes to this file

Reply:
oh, okay 
should I remove this change 
actually that version of ruby was not working for me. so, I changed locally but I pushed it too by mistake

Reply:
@AvikantSrivastava  
I have made all the changes 
can you review my code again